<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DELL-2D26CVB" ignored="0" name="Default test" tests="4" failures="1" timestamp="16 Mar 2022 11:52:06 GMT" time="379.35" errors="0">
  <testcase name="createorganizationTestNGTest" time="65.043" classname="com.crm.OrganizationTests.CreateOrganizationTestNGTest">
    <skipped/>
  </testcase> <!-- createorganizationTestNGTest -->
  <testcase name="createorganizationTestNGTest" time="16.688" classname="com.crm.OrganizationTests.CreateOrganizationTestNGTest">
    <skipped/>
  </testcase> <!-- createorganizationTestNGTest -->
  <testcase name="createorganizationTestNGTest" time="19.076" classname="com.crm.OrganizationTests.CreateOrganizationTestNGTest">
    <skipped/>
  </testcase> <!-- createorganizationTestNGTest -->
  <testcase name="createorganizationTestNGTest" time="23.102" classname="com.crm.OrganizationTests.CreateOrganizationTestNGTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at com.crm.OrganizationTests.CreateOrganizationTestNGTest.createorganizationTestNGTest(CreateOrganizationTestNGTest.java:25)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- createorganizationTestNGTest -->
</testsuite> <!-- Default test -->
